templates/styles/main.css

```
@@ -75,12 +81,27 @@ button.primary {
+.background-container.is-search-mode {
+  height: auto;
+  padding: 24px 48px;
+}
+
+.logo-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  gap: 1.5rem;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)